### ISSUE TITLE
Optimizer created manually outside Module but rescale_grad is not normalized to 1.0/batch_size/num_workers (1.0 vs. 0.00390625)

### ISSUE THREAD

**Issue by roboserg** [awslabs/keras-apache-mxnet#127]
I am a newbie keras user. After switching to mxnet I get following warning:

> UserWarning: Optimizer created manually outside Module but rescale_grad is not normalized to 1.0/batch_size/num_workers (1.0 vs. 0.00390625). Is this intended?
>   force_init=force_init)

Using mxnet 2.1.6.1

Code:

```
def simple_lstm_model(do,
                      maxlen=MAX_LEN,
                      max_words=NUM_WORDS,
                      embed_size=EMBED_SIZE):
    model = Sequential()
    model.add(
        Embedding(
            NUM_WORDS,
            EMBED_SIZE,
            weights=[word2vec_matrix],
            trainable=False,
            input_length=MAX_LEN))
    model.add(Dropout(do))
    model.add(LSTM(64, return_sequences=False, name='lstm_layer', unroll=True))
    model.add(Dense(1, activation='sigmoid'))

    model.compile(
        loss='binary_crossentropy',
        optimizer='adam',
        metrics=['accuracy'],
        context=["gpu(0)"])

    return model

history = model.fit(
        x_text_train, (y_train),
        batch_size=256,
        epochs=5,
        validation_split=0.1,
        verbose=0)
```

**Comment by sandeep-krishnamurthy** [comment]
@roboserg - Thanks for using keras-mxnet.
You could safely ignore the warning.

It is informing that MXNet native optimizer is being used (in MXNet backend).

Resolving the issue. Please file issues if you face any issue.
